src/routes/postSkipSegments.js

```diff
@@ -97,14 +98,78 @@ function sendWebhooks(userID, videoID, UUID, segmentInfo) {
     }
 }
 
-// submission: {videoID, startTime, endTime}
+function sendWebhooksNB(userID, videoID, UUID, startTime, endTime, category, probability) {
+    if (config.youtubeAPIKey !== null) {
+        //let submissionInfoRow = db.prepare('get', "SELECT s.videoID, s.userID, s.startTime, s.endTime, s.category, u.userName, " +
+        //    "(select count(1) from sponsorTimes where userID = s.userID) count, " +
+        //    "(select count(1) from sponsorTimes where userID = s.userID and votes <= -2) disregarded " +
+        //    "FROM sponsorTimes s left join userNames u on s.userID = u.userID where s.userId=?",
+        //[userID]);
+        let submissionCount = db.prepare('get', "SELECT COUNT(*) count FROM sponsorTimes WHERE userID=?", [userID]);
+        let disregardedCount = db.prepare('get', "SELECT COUNT(*) disregarded FROM sponsorTimes WHERE userID=? and votes <= -2", [userID]);
+        //let uName = db.prepare('get', "SELECT userName FROM userNames WHERE userID=?", [userID]);
```

Comment:
Probably would be more efficient to write this as one query/request. `uName` is causing an error. Not sure what's causing it yet.

Reply:
The webhook is called before anything is inserted into the DB. For first time submitters, they won't return any userName so `uName.userName = undefined`.  Not sure when `setUsername.js` is called so I don't know if this is a real issue in prod. In any case, I wrapped the code in a try-catch.

Reply:
Yes, this is a real issue in prod. I added a comment on what could be changed to remove the try catch